diff --git a/ext_src/imgui/imgui.cpp b/ext_src/imgui/imgui.cpp
index 6032988..6e72d1f 100644
--- a/ext_src/imgui/imgui.cpp
+++ b/ext_src/imgui/imgui.cpp
@@ -877,7 +877,7 @@ ImGuiIO::ImGuiIO()
     DisplaySize = ImVec2(-1.0f, -1.0f);
     DeltaTime = 1.0f/60.0f;
     IniSavingRate = 5.0f;
-    IniFilename = "imgui.ini";
+    IniFilename = NULL;
     LogFilename = "imgui_log.txt";
     MouseDoubleClickTime = 0.30f;
     MouseDoubleClickMaxDist = 6.0f;
